content/docs/csidriver/installation/test/certcsi.md

```diff
@@ -62,313 +147,524 @@ storageClasses:
-    ephemeral: # if exists, then run EphemeralVolumeSuite
-      driver: # driver name for EphemeralVolumeSuite
+    RWOP: # set this to enable the execution of the MultiAttachSuite with the AccessMode set to ReadWriteOncePod (true or false)
+    ephemeral: # if exists, then run EphemeralVolumeSuite. See the Ephemeral Volumes suite section for example Volume Attributes
+      driver: # driver name for EphemeralVolumeSuite (e.g., csi-vxflexos.dellemc.com)
       fstype: # fstype for EphemeralVolumeSuite
-      volumeAttributes: # volume attrs for EphemeralVolumeSuite. 
+      volumeAttributes: # volume attrs for EphemeralVolumeSuite.
         attr1: # volume attr for EphemeralVolumeSuite
         attr2: # volume attr for EphemeralVolumeSuite
+    capacityTracking:
+      driverNamespace: # namepsace where driver is installed
+      pollInterval:    # duration to poll capacity (e.g., 2m)  
 ```
 
-### Launching Certification Test Run
+Driver specific examples:
 
-After preparing a certification configuration file, you can launch certification by running 
-```bash
-cert-csi certify --cert-config <path-to-config>
-Optional Params:
-   --vsc: volume snapshot class, required if you specified snapshot capability
-   --timeout: set the timeout value for certification suites
-   --no-metrics: disables metrics aggregation (set if you encounter k8s performance issues)
-   --path: path to folder where reports will be created (if not specified ~/.cert-csi/ will be used)
-```
+   {{< tabs name="cerity-config-examples" >}}
+   {{% tab name="CSI PowerFlex" %}}
+
+```yaml
+storageClasses:
+  - name: vxflexos
+    minSize: 8Gi
+    rawBlock: true
+    expansion: true
+    clone: true
+    snapshot: true
+    RWX: false
+    ephemeral:
+      driver: csi-powerstore.dellemc.com
+      fstype: ext4
+      volumeAttributes:
+        volumeName: "my-ephemeral-vol"
+        size: "8Gi"
+        storagepool: "sample"
+        systemID: "sample"
+  - name: vxflexos-nfs
+    minSize: 8Gi
+    rawBlock: false
+    expansion: true
+    clone: true
+    snapshot: true
+    RWX: true
+    RWOP: true
+    ephemeral:
+      driver: csi-vxflexos.dellemc.com
+      fstype: "nfs"
+      volumeAttributes:
+        volumeName: "my-ephemeral-vol"
+        size: "8Gi"
+        storagepool: "sample"
+        systemID: "sample"
+    capacityTracking:
+      driverNamespace: powerstore
+      pollInterval: 2m
+```
+
+   {{% /tab %}}
+   {{% tab name="CSI PowerScale" %}}
 
-## Functional Tests
+```yaml
+storageClasses:
+  - name: isilon
+    minSize: 8Gi
+    rawBlock: false
+    expansion: true
+    clone: true
+    snapshot: true
+    RWX: false
+    ephemeral:
+      driver: csi-isilon.dellemc.com
+      fstype: nfs
+      volumeAttributes:
+        size: "10Gi"
+        ClusterName: "sample"
+        AccessZone: "sample"
+        IsiPath: "/ifs/data/sample"
+        IsiVolumePathPermissions: "0777"
+        AzServiceIP: "192.168.2.1"
```

Comment:
IP?

Reply:
It's a placeholder copied from the storage class example.

https://github.com/dell/csi-powerscale/blob/3a90ab76d2f1d77190565c0caab4330e75f2dd5b/samples/storageclass/isilon.yaml#L49